admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -322,6 +438,9 @@ Execution example:
    For a more detailed instructions about how to run the upgrade with both
    Elasticsearch instances running in the same machine `visit our Wiki`_.
 
+   Verify that you hve a working ElasticSearch 1.7 instance with all your data
+   before you start the upgrade!
```

Comment:
It might be helpful to include a line about how people can verify this.

Reply:
Instructions for checking that are already in the wiki page, can I create a codeblock inside a note?